### PR TITLE
add tests for reducer/pastIterations.js

### PR DESCRIPTION
**Pull request by felipebrunetti94**
Add test for the pastIterations reducer.

### CODE REVIEW

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          storyIds: [ 1, 2 ]
+        },
+        { iterationNumber: 2}
+      ]);
+    });
+  });
+
+
+  describe("ERROR_REQUEST_PAST_STORIES", () => {
+    it("", () => {
```
Comment (negreirosleo) [inline spec/javascripts/reducers/pastIterations_spec.js]: it "" ?

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import actionTypes from 'actions/actionTypes';
+import reducer from "../../../app/assets/javascripts/reducers/pastIterations";
+
+describe("Past Iterations Reducer", () => {
```
Comment (paulodiovani) [inline spec/javascripts/reducers/pastIterations_spec.js]: There are mixed use of single and double quotes,
Unless the linter is specifying the contrary, I suggest always using single quotes.

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  describe("RECEIVE_PAST_ITERATIONS", () => {
+    it("returns formatted past iterations,", () => {
```
Comment (nolleto) [inline spec/javascripts/reducers/pastIterations_spec.js]: Typo 

```diff
-    it("returns formatted past iterations,", () => {
+    it("returns formatted past iterations", () => {
```

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        storyIds: [],
+        fetched: false,
+        isFetching: false,
+        stories: undefined
+      }]);
+    });
+  });
+
+  describe("REQUEST_PAST_STORIES", () => {
+    it("add isFetching: true, to the selected iteration ", () => {
```
Comment (nolleto) [inline spec/javascripts/reducers/pastIterations_spec.js]: `adds` 💅

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+        iterationNumber: 1,
+        error: Error('whoopsie')
+      };
+
+      expect(reducer(state, action)).toEqual([
+        {
+          iterationNumber: 1,
+          fetched: false,
+          isFetching: false,
+          error: Error('whoopsie')
```
Comment (nolleto) [inline spec/javascripts/reducers/pastIterations_spec.js]: Missing `new`

```suggestion
          error: new Error('whoopsie')
```

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+        },
+        {
+          iterationNumber: 2
+        }
+      ];
+
+      const action = {
+        type: actionTypes.ERROR_REQUEST_PAST_STORIES,
+        iterationNumber: 1,
+        error: Error('whoopsie')
```
Comment (nolleto) [inline spec/javascripts/reducers/pastIterations_spec.js]: Missing `new`

```suggestion
        error: new Error('whoopsie')
```

File: spec/javascripts/reducers/pastIterations_spec.js (hunk 3 above)
Comment (felipebrunetti94) [reply spec/javascripts/reducers/pastIterations_spec.js]: thks :smile:

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+        },
+        {
+          iterationNumber: 2
+        }
+      ];
+
+      const action = {
+        type: actionTypes.ERROR_REQUEST_PAST_STORIES,
+        iterationNumber: 1,
+        error:new Error('whoopsie')
```
Comment (nolleto) [inline spec/javascripts/reducers/pastIterations_spec.js]:
```suggestion
        error: new Error('whoopsie')
```
💅

Comment (felipebrunetti94) [reply spec/javascripts/reducers/pastIterations_spec.js]: whoopsie

File: spec/javascripts/reducers/pastIterations_spec.js [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+      ];
+
+      const action = {
+        type: actionTypes.RECEIVE_PAST_STORIES,
+        iterationNumber: 1,
+        stories:[
+          {id: 1},
+          {id: 2}
+        ],
+        from: undefined
```
Comment (talyssonoc) [inline spec/javascripts/reducers/pastIterations_spec.js]: Do we need this?

Comment (felipebrunetti94) [reply spec/javascripts/reducers/pastIterations_spec.js]: No we don't, but I though make the action from the test more like the real action, would be a good idea. What do you think ?

**Pull request merged**